product_docs/docs/migration_portal/4/03_mp_using_portal/mp_ai_copilot/ai_good_prompts.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: How to create good prompts 
+
+---
+
+The quality and accuracy of the answers the AI Copilot provides depends highly on your prompt. 
+The AI Copilot works best if it has large context around a question, details or specifics about the situation, and examples. 
+The more detailed and precise you can make your prompt, the better answers you will obtain. 
+
+Like with other language models, you can use prompt engineering techniques with the AI Copilot to improve your query input. 
+
+!!! Note Data considerations
+    All prompts inserted in the chatbox are stored in an EDB-managed backend database and stored by Microsoft Azure's OpenAI.
```

Comment:
Should we refer to it more generally as `AI service provider`?  
Also, we could say the data is `processed` rather than stored, but if it is indeed stored, I'd prefer we call that out.

Reply:
Here is a copy of the Microsoft Azure Open AI data privacy page that Mike provided a link to:
https://learn.microsoft.com/en-us/legal/cognitive-services/openai/data-privacy

According to that page, by default prompts and responses are stored for up to 30 days in support of their abuse monitoring capabilities. So, I think it is ok to say that the prompts inserted in the chatbox are stored by the AI service provider.